src/Symfony/Component/ExpressionLanguage/Tests/LexerTest.php

```diff
@@ -127,6 +127,25 @@ public function getTokenizeData()
                 ],
                 'foo.not in [bar]',
             ],
+            [
+                [new Token('number', '0.787', 1)],
+                '0.787',
+            ],
+            [
+                [new Token('number', '0.1234', 1)],
+                '.1234',
+            ],
+            [
+                [new Token('number', '188165.1178', 1)],
```

Comment:
@stof There are two tests here using `_`, did you think about another case?

Reply:
ah sorry, I looked at it too fast